pennylane/ops/qubit/parametric_ops.py

```diff
@@ -1269,6 +1269,182 @@ def control_wires(self):
         return self.wires[0:1]
 
 
+class PCPhase(Operation):
+    r"""
+    A Projector Controlled Phase gate.
+
+    .. math:: \Pi_{\phi}(\phi, dim=2, wires=[0,1]) = \begin{bmatrix}
+                e^{i\phi} & 0 & 0 & 0 \\
+                0 & e^{i\phi} & 0 & 0 \\
+                0 & 0 & 1 & 0 \\
+                0 & 0 & 0 & 1
+            \end{bmatrix}.
+
+    Args:
+        phi (float): rotation angle :math:`\phi`
+        dim (int): the qubit subspace on which we apply the phase shift
+        wires (Any, Wires): the wire the operation acts on
+        do_queue (bool): Indicates whether the operator should be
+            immediately pushed into the Operator queue (optional)
+        id (str or None): String representing the operation (optional)
+
+    **Details:**
+
+    * Number of wires: AnyWires
+    * Number of parameters: 1
+    * Number of dimensions per parameter: (0,)
+    * Gradient recipe: (~ ADD THIS ~)
+
+    **Example:**
+
+    >>> pc_op = qml.PCPhase(1.23, dim=3, wires=[1,2])
+    >>> qml.matrix(pc_op)
+    array([[0.33423773+0.9424888j, 0.        +0.j       , 0.        +0.j       , 0.        +0.j       ],
+           [0.        +0.j       , 0.33423773+0.9424888j, 0.        +0.j       , 0.        +0.j       ],
+           [0.        +0.j       , 0.        +0.j       , 0.33423773+0.9424888j, 0.        +0.j       ],
+           [0.        +0.j       , 0.        +0.j       , 0.        +0.j       , 1.        +0.j       ]])
+    """
+    num_wires = AnyWires
+    num_params = 1
+    """int: Number of trainable parameters that the operator depends on."""
+    ndim_params = (0,)
+    """tuple[int]: Number of dimensions per trainable parameter that the operator depends on."""
+
+    basis = "Z"
+    grad_method = "A"
+    parameter_frequencies = [(1,)]
+
+    def generator(self):
+        r"""The hermitian matrix, which when exponentiated and scaled (i \phi), produces
```

Comment:
The `(i \phi)` equation doesn't render correctly.

Reply:
Added 👍🏼